src/box/iproto.cc

```diff
@@ -3664,6 +3665,8 @@ iproto_thread_init_routes(struct iproto_thread *iproto_thread)
 	iproto_thread->dml_route[12] = NULL;
 	/* IPROTO_PREPARE */
 	iproto_thread->dml_route[13] = iproto_thread->sql_route;
+	/* IPROTO_INSERT_ARROW */
+	iproto_thread->dml_route[17] = iproto_thread->process1_route;
```

Comment:
Using numbers here looks really ugly. Let's replace them with `IPROTO_XXX` constants while we are at it. Okay to do in a separate commit.

Reply:
Done.